helper-cli/src/main/kotlin/commands/ListPackagesCommand.kt

```diff
@@ -74,3 +81,5 @@ class ListPackagesCommand : CliktCommand(
         println(result)
     }
 }
+
+private enum class Type { PROJECT, PACKAGE }
```

Comment:
How about moving

https://github.com/oss-review-toolkit/ort/blob/c60ee6232727025c8fb0db2aa1af2465923228a8/cli/src/main/kotlin/OrtMain.kt#L60-L73

to a common place for you to reuse it?

Reply:
Hm, I'm struggling to find a good place for this. To me this class seems quite CLI specific, so I hesitate to move it out of the `cli` module. Do you have any place in mind where it can be put? (and how it could be named)

Reply:
I'd simply put it in its own file under the `model` module, like the `Package` and `Project` it refers to.